packages/app-project/src/components/ProjectHeader/ProjectHeader.stories.js

```diff
@@ -0,0 +1,320 @@
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
+import { Provider } from 'mobx-react'
+import { applySnapshot } from 'mobx-state-tree'
+import { RouterContext } from 'next/dist/shared/lib/router-context'
+
+import initStore from '@stores'
+import ProjectHeader from './ProjectHeader.js'
+import * as subcomponents from './components'
+
+const mockRouter = {
+  asPath: '/zooniverse/snapshot-serengeti/about/team',
+  basePath: '/projects',
+  locale: 'en',
+  push(url, unused, options) {
+    pageURL = url
+    locale = options.locale
+  },
```

Comment:
What feature does this mock? I'm curious why `push()` has `locale` when locale is already a key in the router object.

Reply:
The menu items in the language menu call `router.push` when they’re clicked. I think that was a workaround for a bug that’s since been fixed in `next-i18n`, where the page reloads when the locale changes. 

https://github.com/zooniverse/front-end-monorepo/blob/master/packages/app-project/src/components/ProjectHeader/components/LocaleSwitcher/LocaleSwitcher.js#L37-L40

I think I can get rid of these lines, because they don’t get called when the links are clicked in the tests.